pkg/api/v1/helper/helpers.go

```diff
@@ -14,21 +14,28 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// NOTE: DO NOT use those helper functions through client-go, the
```

Comment:
I haven't made my mind for pkg/api/v1/helper, we might move all the v1 helper function from kubernetes to client-go, so let's only comment in pkg/api/helper